frontend/src/components/common/datepicker/datepicker.tsx

```diff
@@ -1,26 +1,22 @@
 import 'react-datepicker/dist/react-datepicker.css';
 import './styles.scss';
 
-import {
-  FC,
-  FormControl,
-  FormControlErrors,
-  FormControlPath,
-} from 'common/types/types';
-import { ErrorMessage } from 'components/common/common';
+import { FC, FormControl, FormControlPath } from 'common/types/types';
+import { subYears } from 'date-fns';
```

Comment:
```suggestion
```
we have this package in the shared moduel

Reply:
we need to create a new helper in the `shared` folder and then use it
we don't need to hardcode this value in the common component